frontend/packages/integration-tests-cypress/cy-command-additions/login.ts

```diff
@@ -0,0 +1,51 @@
+import { submitButton } from '../utils/selectors';
+
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace,no-redeclare
+  namespace Cypress {
+    interface Chainable<Subject> {
+      login(providerName?: string, username?: string, password?: string): Chainable<Element>;
+      logout(): Chainable<Element>;
+    }
+  }
+}
+
+const usernameInput = '#inputUsername';
+const passwordInput = '#inputPassword';
```

Comment:
If these are only used in one place, just inline the values.

Reply:
There are test still left to implement for login, I imagine these const will be used multiple times.

Reply:
> There are test still left to implement for login, I imagine these const will be used multiple times.

We can extract a constant when we need it. For now, it's better to inline.

Reply:
fixed